src/components/custom/other-platforms.js

```diff
@@ -224,226 +224,289 @@ OtherPlatform.propTypes = {
     subHeader: PropTypes.string,
 }
 
-export const NavPlatform = ({ onClick }) => (
-    <Flex>
+export const NavPlatform = ({ onClick, setCurrentPage }) => {
+    const links = {
+        dtrader: '/dtrader',
+        dbot: '/dbot',
+        dmt5: '/dmt5',
+        smarttrader: 'trading',
+        margin: '/trade-types/margin',
+        options: '/trade-types/options',
+        multipliers: '/trade-types/multiplier',
+    }
+
+    useEffect(() => {
+        if (Object.values(links).includes(window.location.pathname)) setCurrentPage('trade')
+    }, [])
```

Comment:
I think you'll need to add a dependency here to check when the hash changes. Cause right now, if I were to key in `/about` it will automatically reroute to `/about#story` without updating the state, hence not showing the active bar. Same goes for all the rest.

Reply:
Good catch~ I'll check this out. Thanks!